new/LIT6193HomiliaryPassions.xml

```diff
@@ -0,0 +1,155 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
```

Comment:
was there an issue opened to discuss the new record? did I miss it?

Reply:
Yes, there was an issue, some time ago, initiated by Ralph.